src/de/2024-04/03/teacher-questions.md

```diff
@@ -86,8 +94,10 @@ date:   18/10/2024
     Vers 17 das erste Mal)
 -   *Was berührt euch ganz persönlich an Johannes 1,1-18?*
 
-*The message of John: here is your king!: with study guide*, The Bible
-Speaks Today (Leicester, England; Downers Grove, IL: InterVarsity Press,
-1993), 4
+----[^(\[1\])](4/2024_Lektion_03#_ftnref1 "wikilink") Bruce Milne, *The
+message of John: here is your king!: with study guide*, The Bible Speaks
+Today (Leicester, England; Downers Grove, IL: InterVarsity Press, 1993),
+4
 
-Weitere Fragen oder Ideen? - du bist herzlich zur Mitarbeit eingeladen: https://wiki.sabbatschule.at
+Weitere Fragen oder Ideen? - du bist herzlich zur Mitarbeit eingeladen:
+<https://wiki.sabbatschule.at>
```

Comment:
@toak Can you please remove surrounding `<` and `>`

Reply:
will always be removed